src/controllers/organisms/RoomView.js

```diff
@@ -346,7 +359,7 @@ module.exports = {
                 }
             }
             ret.unshift(
-                <li key={mxEv.getId()}><EventTile mxEvent={mxEv} continuation={continuation} last={last}/></li>
+                <li ref={this._collectEventNode.bind(this, mxEv.getId())} key={mxEv.getId()}><EventTile mxEvent={mxEv} continuation={continuation} last={last}/></li>
```

Comment:
refs as strings is apparently kind of deprecated in react now, and they should be functions that get called with the react node once it gets mounted.